classes/class-wc-connect-service-settings-store.php

```diff
@@ -365,8 +365,8 @@ public function get_enabled_services_by_ids( $service_ids ) {
 				$service_settings = $this->get_service_settings( $method->method_id, $method->instance_id );
 				if ( is_object( $service_settings ) && property_exists( $service_settings, 'title' ) ) {
 					$title = $service_settings->title;
-				} else if ( is_object( $service_schema ) && property_exists( $service_schema, 'method_title' ) ) {
-					$title = $service_schema->method_title;
+				} else if ( is_object( $service_schema ) && property_exists( $service_schema, 'carrier_name' ) ) {
```

Comment:
Where is this used? I can see the function `get_enabled_services` is called in the helper, but I don't see any mention of the service names on the helper page.

Reply:
In the Self-Help page (`WooCommerce -> Status -> WooCommerce Services`) all the WCS configured methods are listed, and for each one it shows the shipping method name (its title), last successful rate fetch timestamp, etc. Here: https://github.com/Automattic/woocommerce-services/blob/ee8b7445c3d8d4b1d893d7938ab6f0ec0553b5d8/classes/class-wc-connect-help-view.php#L202